Source/UIKit additions/SVGKFastImageView.m

```diff
@@ -130,20 +130,26 @@ - (void)setImage:(SVGKImage *)image {
     }
     
     /** other obeservers */
+  if (!self.didResigesterObservers) {
+    self.didResigesterObservers = true;
     [self addObserver:self forKeyPath:@"image" options:NSKeyValueObservingOptionNew context:internalContextPointerBecauseApplesDemandsIt];
     [self addObserver:self forKeyPath:@"tileRatio" options:NSKeyValueObservingOptionNew context:internalContextPointerBecauseApplesDemandsIt];
     [self addObserver:self forKeyPath:@"showBorder" options:NSKeyValueObservingOptionNew context:internalContextPointerBecauseApplesDemandsIt];
+  }
+
 }
 
 -(void) addInternalRedrawOnResizeObservers
 {
+  if (self.didResigesterInternalRedrawObservers) return;
+  self.didResigesterInternalRedrawObservers = true;
 	[self addObserver:self forKeyPath:@"layer" options:NSKeyValueObservingOptionNew context:internalContextPointerBecauseApplesDemandsIt];
 	[self.layer addObserver:self forKeyPath:@"transform" options:NSKeyValueObservingOptionNew context:internalContextPointerBecauseApplesDemandsIt];
-	//[self.image addObserver:self forKeyPath:@"size" options:NSKeyValueObservingOptionNew context:internalContextPointerBecauseApplesDemandsIt];
 }
 
 -(void) removeInternalRedrawOnResizeObservers
 {
+  if (!self.didResigesterInternalRedrawObservers) return;
```

Comment:
This method should ALSO set .didRegisterInternalRedrawObservers back to FALSE at the end, no?